src/index.ts

```diff
@@ -1,3 +1,3 @@
-export { GenericContainer } from "./generic-container";
+export { GenericContainer, GenericContainerBuilder } from "./generic-container";
```

Comment:
What's the use case for exposing this?

Reply:
When you want to temporary store the builder somewhere and you want to have type-safety you need to export the builder type.